volumina/croppingMarkers.py

```diff
@@ -65,21 +84,35 @@ def set_crop_extents(self, crop_extents):
         self._crop_extents = map(list, crop_extents) # Ensure lists, not tuples
         self.changed.emit( self )
 
+    def cropZero(self):
+        if self._crop_extents == None:
+            return True
+
+        flag = True
+        for c in self._crop_extents:
+            flag = (flag and c[0]==0 and c[1]==0)
+
+        return flag
+
 class CroppingMarkers( QGraphicsItem ):
     PEN_THICKNESS = 1
 
     def boundingRect(self):
-        # Return an empty rect to indicate 'no content'
-        # This 'item' is merely a parent node for child items
-        return QRectF()
+        width, height = self.dataShape
+        return QRectF(0.0, 0.0, width, height)
+
+    def __init__(self, scene, axis, crop_extents_model, editable=True):
+
+        self._cropColor = Qt.white
+        self._editable = editable
 
-    def __init__(self, scene, axis, crop_extents_model):
         QGraphicsItem.__init__(self, scene=scene)
-        self.setFlag(QGraphicsItem.ItemHasNoContents);
-        
```

Comment:
I might be missing something here, but I don't understand why you remove the `ItemHasNoContents` flag here.  Below, you provided an implementation of `paint()`, so why not set the flag here?